lib/fluent/plugin/output.rb

```diff
@@ -310,6 +310,10 @@ def configure(conf)
             log.warn "'flush_at_shutdown' is false, and buffer plugin '#{buf_type}' is not persistent buffer."
             log.warn "your configuration will lose buffered data at shutdown. please confirm your configuration again."
           end
+
+          if @flush_mode != :interval && buffer_conf.has_key?('flush_interval')
```

Comment:
> It looks better to set :interval when flush_mode is :default

https://github.com/fluent/fluentd/blob/master/lib/fluent/plugin/output.rb#L293

From above line, `:default` is changed to `:lazy` or `:interval` so `:default` is not appeared here, right?

>  how about to add has_flush_interval before super as same with has_buffer_section?

It seems good.

> these are exclusive parameters

If so, should we raise an ConfigError when `flush_interval` and `flush_mode lazy` are set explictly?